### ISSUE TITLE
Make Location Based examples use absolute paths on import scripts

### ISSUE THREAD

**Issue by nicolocarpignoli** [jeromeetienne/AR.js#605]
Actually, on both https://github.com/jeromeetienne/AR.js/tree/master/aframe/examples/click-places and https://github.com/jeromeetienne/AR.js/tree/master/aframe/examples/places-name, on index.html files, aframe components of Location Based AR.js are imported with relative paths. 

This works, as we are on AR.js repo, but in order to make more useful and instructional examples, it will be better to replace them with absolute path to files from AR.js repo, retrieved from https://raw.githack.com, for example, as it's already done for the AR.js lib.

Basically, those imports needs to be changed:

```
<!-- include GeoAR.js -->
<script src="../../src/location-based/gps-entity-place.js"></script>
<script src="../../src/location-based/gps-camera-debug.js"></script>
<script src="../../src/location-based/gps-camera.js"></script>
<link rel="stylesheet" href="../../src/location-based/stylesheets/index.css">
```
